components/match2/wikis/valorant/big_match.lua

```diff
@@ -233,9 +237,15 @@ function BigMatch:economy(match, opponent1, opponent2)
 
 		tabs['name' .. ind] = 'Map ' .. ind
 
-		local data = {}
-		for index, round in ipairs(map.rounds) do
-			table.insert(data, {
+		local function processRound(index, storage)
```

Comment:
Why are these functions local?

Reply:
Made them private class functions instead of local functions